io.github.tomluchowski.OpenDungeonsPlus.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Comment=OpenDungeonsPlus, a better fork of Opendungeons
+Categories=Game;Strategy;
+Exec=opendungoens-plus
+Icon=io.github.tomluchowski.OpenDungeonsPlus.desktop
```

Comment:
this icon name is invalid. I think you are confusing icon name with .desktop file name.

Reply:
sure, does it expects absolute path or just the filename ?

Reply:
`Icon` is a name not a file. The name is then looked up in various paths.